src/treeinterp/BTFactory.cpp

```diff
@@ -4,12 +4,20 @@
 
 #include "treeinterp/BTFactory.h"
 
+// The tactics variable saves the tactic of the behaviour tree and can be kept
+
+// Changes made by crazy person:
```

Comment:
Please refer from making comments like this, they do not help at all

Reply:
Yes that comment was mostly for me to keep track of where I made changes :grinning: